dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/DiscoverFacetsConverter.java

```diff
@@ -75,6 +85,28 @@ private void addFacetValues(final DiscoverResult searchResult, final SearchResul
         }
     }
 
+    private void handleExposeMinMaxValues(Context context,DiscoverySearchFilterFacet field,
+                                          SearchFacetEntryRest facetEntry) {
+        try {
+            String minValue = searchService.calculateExtremeValue(context,
+                                                                  field.getIndexFieldName() + "_min",
+                                                                  field.getIndexFieldName() + "_min_sort",
+                                                                  DiscoverQuery.SORT_ORDER.asc);
+            String maxValue = searchService.calculateExtremeValue(context,
+                                                                  field.getIndexFieldName() + "_max",
+                                                                  field.getIndexFieldName() + "_max_sort",
+                                                                  DiscoverQuery.SORT_ORDER.desc);
+
+            if (StringUtils.isNotBlank(minValue) && StringUtils.isNotBlank(maxValue)) {
+                facetEntry.setMinValue(minValue);
+                facetEntry.setMaxValue(maxValue);
+            }
+        } catch (SearchServiceException e) {
+            e.printStackTrace();
```

Comment:
Just out of curiosity, I've seen a few `printStackTrace()` calls in this PR...is there a reason we aren't logging these errors (e.g. `log.error();`)?

Reply:
No, they should indeed be logged using `log.error()`.